src/field_system_weather.c

```diff
@@ -15,19 +15,18 @@ const MonthDay DiamondDustDates[] = {
     { DECEMBER, 31 },
 }; //_020FA09C
 
-u32 Fsys_GetWeather_HandleDiamondDust(FieldSystem* fsys, u32 mapId) {
+u32 FieldSystem_GetWeather_HandleDiamondDust(FieldSystem *fieldSystem, u32 mapId) {
```

Comment:
why did you change fsys to fieldSystem?

Reply:
consistency with diamond, literally no other reason

Reply:
we use fsys here bucko

Reply:
fsys is dumb, fieldsystem is better, the struct is named fieldsystem, unless you want me to change that to struct FSys :)

Reply:
you're dumb >=(